src/esm_tests/repos.py

```diff
@@ -1,14 +1,19 @@
 import os
+import questionary
 
 from loguru import logger
 
 from .test_utilities import sh
+from esm_parser import user_error
```

Comment:
We might consider moving this into `esm_utilities` as is it used often enough everywhere else. It isn't really a parsing thing.

Reply:
Yeap, I agree, but that's a lot of changes in many places. I think this topic will come up again when we cleanup the parser, so we can leave it for that time?

Reply:
yes. in any case that will be a separate PR, and should not be done here.